contributors/guide/OWNERS

```diff
@@ -1,13 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-reviewers:
-  - castrojo
-  - guineveresaenger
-  - idvoretskyi
-  - tpepper
 approvers:
-  - castrojo
+  - mrbobbytables
   - parispittman
+emeritus_approvers:
+  - castrojo
   - guineveresaenger
```

Comment:
@castrojo , @guineveresaenger 

Would you like to remain as emeritus approvers in the contributor guide owners file?